internal/driver/glfw/window_test.go

```diff
@@ -821,6 +821,49 @@ func TestWindow_Clipboard(t *testing.T) {
 	cb.SetContent(cliboardContent)
 }
 
+func TestWindow_WillClose(t *testing.T) {
+	closed := make(chan bool)
+
+	createLocaWindow := func() fyne.Window {
+		w := createWindow("Test")
+		w.SetCloseIntercept(func() {
+			go func() {
+				closed <- true
+				w.Close()
+			}()
+		})
+		return w
+	}
+
+	checkClosed := func() bool {
```

Comment:
Maybe this should be 'checkIntercepted'? It's actually checking that the window /didn't/ automatically close.